src/server/scripts/Pet/pet_mage.cpp

```diff
@@ -28,6 +28,8 @@
 #include "Pet.h"
 #include "PetAI.h"
 #include "ScriptedCreature.h"
+#include "SpellHistory.h"
```

Comment:
how is any change in this file "dancing rune weapon"?

Reply:
cause I used it as reference

Reply:
and this is a test build pr pretty much 💃